### PR TITLE
add TextArea component

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are adding new component `TextArea`.

### CODE REVIEW

File: src/components/TextArea/TextArea.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        defaultValue=""
+        error="that's an error with|without content"
+      />
+      <br />
+      <TextArea
+        placeholder={'something'}
+        defaultValue=""
+        warning="that's a warning with|without content"
+      />
+    </div>
```
Comment (mazmassa) [inline src/components/TextArea/TextArea.stories.tsx:54]:
```suggestion
```

File: src/components/TextArea/TextArea.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { TextArea } from './TextArea';
+
+export default { title: 'Components/TextArea' };
+
+export const _TextArea = {
+  render: () => (
+    <div className="bg-secondary text-f-primary p-4">
+      <h3>Controlled components</h3>
+      <br />
```
Comment (mazmassa) [inline src/components/TextArea/TextArea.stories.tsx:9]:
```suggestion
```

File: src/global.css [hunk 3]
```diff
@@ -82,4 +82,19 @@
   .default-tab-active {
     @apply mas-menu-active antialiased hover:border-brand border-brand border-b-2;
   }
+
+  .default-text-area {
+    @apply text-tertiary rounded-lg border resize-none;
+  }
```
Comment (mazmassa) [inline src/global.css:88]: Question: can't we use the same globals as we have in the input ones!?

File: src/components/TextArea/TextArea.stories.tsx (hunk 1 above)
Comment (maxime-carabina) [reply src/components/TextArea/TextArea.stories.tsx:54]: Done

**Pull request merged**